### PR TITLE
split up styles a bit

### PR DESCRIPTION
**Pull request by wheresrhys**

### CODE REVIEW

**Comment by adgad** [comment]
It looks okay to me. One thing I'm wondering is what happens if there's duplication in the individual stylesheets?

i.e. hypothetically if `components/teasers` and `components/n-image` both included say o-grid, would they be created separately and then concat'ed inline? Which means it would output two o-grids?

Generally I've been checking /world, /news-feed, fastft, random tag stream, life and arts and /special-reports to QA.

File: client/main.scss [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+@include nUiStylesheetStart('o-expander');
+	$o-expander-is-silent: false;
+	@import 'o-expander/main';
+@include nUiStylesheetEnd('o-expander');
+
+@include nUiStylesheetStart('search');
+	@import 'components/search/main';
+@include nUiStylesheetEnd('search');
+
+@include nUiStylesheetStart('photo-diary');
```
Comment (adgad) [inline client/main.scss]: Any idea where these actually exist on the site? I can't seem to find them (so possibly a place for further optimisation!)

Comment (wheresrhys) [inline client/main.scss]: I'll have a dig around

**Comment by wheresrhys** [comment]
> what happens if there's duplication in the individual stylesheets?

Good question. It's a risk/consideration when splitting head/main css too. In general duplication of styles is to be avoided given our current tooling for css splitting, and we are being quite good at doing that I think.

This PR seems to not break anything. I'll try a prod build locally to double check

**Comment by i-like-robots** [comment]
Have checked this out and it builds fine. All looks good except for a slightly naughty right rail.

**Comment by wheresrhys** [comment]
closing in favour of https://github.com/Financial-Times/next-stream-page/pull/1721 as circle is being weird

**Pull request closed without merging**